README.md

```diff
@@ -17,3 +17,36 @@ Setup a systemd timer that cleans up unused and dangling images every night by r
 ### docker.compose
 
 Install `docker-compose` package.
+
+### docker.pillar-users
+
+Populate the `docker` group with a list of usernames extracted from Salt pillar data.
+The pillar of the targeted minion should contain the following, where each subkey (`FIRSTSET`, `SECONDSET`, ...) and associated list of users may originate from a different pillar file targeted to the minion (e.g. due to overlapping wildcard matches in pillar targeting).
+
+```
+docker-users:
```

Comment:
This should be named `docker.pillar_users` (maybe even just `docker.users` - given that we should remove the user state and configure all users by pillars).